server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SetElectionInfo.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 9, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @author Daniel M. Zimmerman <dmz@galois.com>
+ * @description A system to assist in conducting statewide risk-limiting audits.
+ */
+
+package us.freeandfair.corla.endpoint;
+
+import javax.persistence.PersistenceException;
+
+import com.google.gson.JsonParseException;
+
+import spark.Request;
+import spark.Response;
+
+import us.freeandfair.corla.Main;
+import us.freeandfair.corla.json.SubmittedElectionInfo;
+import us.freeandfair.corla.model.DoSDashboard;
+import us.freeandfair.corla.persistence.Persistence;
+
+/**
+ * The endpoint for setting the election information.
+ * 
+ * @author Daniel M Zimmerman
+ * @version 0.0.1
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
+public class SetElectionInfo extends AbstractDoSDashboardEndpoint {
+  /**
+   * The "random seed" parameter.
+   */
+  public static final String RANDOM_SEED = "random_seed";
```

Comment:
Copy-pasta?

Reply:
I'm not certain what you are saying here...

Reply:
Was this accidentally copy-pasted? It doesn't seem to be used anywhere in this class.

Reply:
Ah, now I see. I missed what class it was on.